packages/pipeline-editor/src/NotebookSubmissionDialog.tsx

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { IDictionary } from '@elyra/application';
+import * as React from 'react';
+
+import { IRuntime } from './PipelineEditorWidget';
+
+type Props = {
```

Comment:
is there a reason you're using `type` for Props throughout this PR instead of `interface`? As far as jupyterlab (and elyra) code goes it seems to always use `interface` for props

Reply:
I normally always use `interface` too, not sure what led me to use `type` here haha